india_stocks_api/internal/fivepaisa/api/order_api.py

```
@@ -0,0 +1,627 @@
+                "symbol": symbol,
+                "action": action,
+                "exchange": exchange,
+                "pricetype": "MARKET",
+                "product": reverse_map_product_type(position['OrderFor'],exchange),
+                "quantity": str(quantity)
+            }
+
+            logger.info(f"{place_order_payload}")
+
+            # Place the order to close the position
+            res, response, orderid =   place_order_api(place_order_payload,auth)
```

Ruff (0.14.14)

Unpacked variable `res` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)